posts/bonus-pokedex.md

```diff
@@ -0,0 +1,392 @@
+# How to build a Pokedex with Platformatic
+
+Recently, [Matteo Collina](https://nodeland.dev/), one of Fastify's creators and much more, launched a new Backend As A Service service: [Platformatic](https://platformatic.dev/).
+Since it is built on top of Fastify, it claims to be a life changer, and I want to try it and write down my thoughts!
+
+So, what are we going to build? A Pokedex!  
+Of course, I don't want to write a boring article, let's make it a bit more fun!  
+Here are the requirements:
+
+- The Pokemon database already exists. It will be our "legacy" system
+- I want to build a React frontend. It will test Platformatic's "extensibility". Note that Platformatic is a "Backend" framework, but fastify can handle frontend too!
+- The application must be read-only. I don't want to expose any API with `write` privileges.
+
+Now that we have the requirements let's start!
```

Comment:
A small architecture diagram would be great!

Reply:
done

<img width="553" alt="image" src="https://user-images.githubusercontent.com/11404065/218317698-f5f44d1a-6c17-4650-bafe-6b03512ff06e.png">

Reply:
is the "React Application Build" a database?

Reply:
it is the file system - I will try to draw it in a clearer way